assets/js/blocks/cart-checkout/cart/full-cart/cart-line-items-table.js

```diff
@@ -3,16 +3,30 @@
  */
 import { __ } from '@wordpress/i18n';
 import PropTypes from 'prop-types';
+import { useMemo } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import CartLineItemRow from './cart-line-item-row';
 
-const CartLineItemsTable = ( { lineItems = [] } ) => {
-	const products = lineItems.map( ( lineItem ) => {
-		return <CartLineItemRow key={ lineItem.key } lineItem={ lineItem } />;
-	} );
+const CartLineItemsTable = ( { lineItems = [], isLoading = false } ) => {
+	const placeholderRows = useMemo( () => {
+		return [ ...Array( 3 ) ].map( ( _x, i ) => (
+			<CartLineItemRow key={ i } />
+		) );
+	}, [] );
```

Comment:
I don't think this needs to be a function, it could be a regular const assignment.
```JS
	const placeholderRows = [ ...Array( 3 ) ].map( ( _x, i ) => (
		<CartLineItemRow key={ i } />
	) );
```
If we are worried about performance on re-renders (I don't think we should, but just in case), we could declare `placeholderRows` outside `CartLineItemsTable`, so it doesn't get called on each render. But that seems premature optimization from my point of view.